### PR TITLE
refactor: remove GCP functionality

### PR DESCRIPTION
**Pull request by tjsilver**
## What does this change?

Removes the GCP functionality from the app/UI.

Ran the app locally and GCP findings and links to them are now gone.

## What is the value of this?

This functionality is currently broken so it shouldn't make any difference to users.
We are paring down Security HQ and migrating functionality to other repos where possible. This will allow us to keep this repo up-to-date more easily.

## Will this require CloudFormation and/or updates to the AWS StackSet?

No.

## Will this require changes to config?

Only the app config - where GCP functionality has been deleted.

### CODE REVIEW

File: hq/app/config/Config.scala [hunk 1]
```diff
@@ -67,20 +67,6 @@ object Config {
     )
   }
 
-  def gcpSccAuthentication(implicit config: Configuration): GcpSccConfig = {
-    val gcpOrgId = requiredString(config, "gcp.orgId")
-    val gcpSccSourceId = requiredString(config, "gcp.sccSourceId")
-    GcpSccConfig(gcpOrgId, gcpSccSourceId)
-  }
-
-  def gcpCredentialsProvider(implicit config: Configuration): FixedCredentialsProvider = {
```
Comment (tjsilver) [inline hq/app/config/Config.scala:76]: @mchv we will need to remove the credentials from the Google Developer Console - I believe you have the permissions to do this?

**Review by NovemberTang — COMMENTED** [review]
I think as a result of this we can remove some dependencies. Namely the `google.cloud.securitycenter` one, but there may be one or two others

**Review by NovemberTang — APPROVED** [review]
Hooray!!!

File: hq/app/config/Config.scala (hunk 1 above)
Comment (tjsilver) [reply hq/app/config/Config.scala:76]: @mchv The credentials can now be deleted - thank you!

**Pull request merged**